tests/utils/constants/constants.go

```diff
@@ -33,6 +33,7 @@ var (
 	WorkSpace           = "WORKSPACE"
 	RHCSPrefix          = "rhcs"
 	TFYAMLProfile       = "tf_cluster_profile.yml"
+	LdapURL             = "ldap://ldap.forumsys.com/dc=example,dc=com?uid"
```

Comment:
This url points to an external LDAP resource, for manual testing every once in a while I assume this should be fine. For automation we should probably find/create an internal LDAP resource, as forumsys.com might go down or block repeated request from us if we start hammering them.

Reply:
true, I added the /hold also for that purpose 
@xueli181114 - I see that in ocm-backend repo you are using `myldap.com` or `forumsys.com`
do you think this is a solution or do we have other option? creating new ldap sever seems to be complex